locals.tf

```
@@ -20,6 +20,10 @@ locals {
       persistence = {
         defaultClass             = var.enable_pv_backups && var.set_default_storage_class ? "false" : "true"
         defaultClassReplicaCount = var.replica_count
+        recurringJobSelector = {
+          enable  = var.automatic_filesystem_trim.enabled && !var.set_default_storage_class ? "true" : "false"
```

```suggestion
          enable  = tostring(var.automatic_filesystem_trim.enabled && !var.set_default_storage_class)
```

Do you need the ternary operator here? Is it because the chart expects a string? We could do something like my suggestion, no?

---

Indeed that should be working, I'll test that